src/simulator/simulator.hpp

```diff
@@ -269,8 +269,16 @@ void Simulator<dim, _interp, nbRefinedPart>::hybrid_init(initializer::PHAREDict
     multiphysInteg_->registerAndSetupMessengers(messengerFactory_);
 
     // hard coded for now, should get some params later from the dict
-    auto hybridTagger_ = amr::TaggerFactory<PHARETypes>::make("HybridModel", "default");
-    multiphysInteg_->registerTagger(0, maxLevelNumber_ - 1, std::move(hybridTagger_));
+    if (dict["simulation"]["AMR"]["refinement"].contains("tagging"))
```

Comment:
I found it weird that we create the Tagger even though we may not be tagging.
And with this new PR it would not work anyway (because the Tagger now takes the threshold from the dict it's now given, and there would not be any threshold in that dict if tagging is not specified by the user)